docs/src/conservation.md

```diff
@@ -13,4 +13,5 @@ ClimaCoupler.ConservationChecker.EnergyConservationCheck
 ClimaCoupler.ConservationChecker.WaterConservationCheck
 ClimaCoupler.ConservationChecker.check_conservation!
 ClimaCoupler.ConservationChecker.plot_global_conservation
+ClimaCoupler.ConservationChecker.surface_water_gain_from_rates
```

Comment:
This function looks like it's internal to the ConservationChecker and not exported, so maybe it would be good to put it under a section `# ConservationChecker Internal Functions` in the docs

Reply:
Good catch!